deploy/helm/sumologic/values.yaml

```diff
@@ -2016,24 +1985,7 @@ kube-prometheus-stack:
             - action: keep
               regex: node-exporter;(?:node_load1|node_load5|node_load15|node_cpu_seconds_total)
               sourceLabels: [job, __name__]
```

Comment:
Since we're dropping recording rule metrics, we need to forward node metrics which are used to create them, so users can reproduce this in Sumo. After this change, we should check that Prometheus sends these metrics: https://github.com/SumoLogic/sumologic-kubernetes-collection/blob/70032194c87cef8bce5aff3eb23d15946433db8a/tests/integration/internal/constants.go#L183 as well.

Reply:
hmm ok, I'll add these to the list of node metrics

Reply:
![Screenshot 2023-08-17 at 11 11 26 AM](https://github.com/SumoLogic/sumologic-kubernetes-collection/assets/113392743/da83b4ce-a85e-4cdf-8354-b3f3072a168a)

Looks like the new node metrics are now being ingested with the latest change

Reply:
Ah, you also need to add these to the ServiceMonitor definition under `additionalServiceMonitors`.

Reply:
ok will take a look. In the meanwhile, the OT metrics test seems to find these unexpected metrics

[kubelet_runtime_operations_duration_seconds_bucket coredns_dns_request_duration_seconds_bucket apiserver_request_duration_seconds_bucket]

Reply:
> Ah, you also need to add these to the ServiceMonitor definition under additionalServiceMonitors

why do we need to scrape these node metrics twice?

Reply:
> ok will take a look. In the meanwhile, not sure why this IT is failing, those `bucket` metrics aren't being collected or expected
> 
> ```
> Test_Helm_OT_Metrics/metrics/expected_metrics_are_present 2023-08-17T21:28:13Z retry.go:103: WaitUntilExpectedMetricsPresent() returned an error: found the following unexpected metrics: [kubelet_runtime_operations_duration_seconds_bucket apiserver_request_duration_seconds_bucket coredns_dns_request_duration_seconds_bucket]. Sleeping for 3s and will try again.
>     assess_funcs.go:202: 'WaitUntilExpectedMetricsPresent()' unsuccessful after 40 retries
> ```

It's an unrelated failure that #3192 fixes.

> > Ah, you also need to add these to the ServiceMonitor definition under additionalServiceMonitors
> 
> why do we need to scrape these node metrics twice?

You don't, my mistake. I got confused for a second there. You should see the node metrics after adding them to the node remotewrite.

Reply:
yes they've been added to the node remotewrite